### ISSUE TITLE
Allow service accounts and custom credentials

### ISSUE THREAD

**Issue by DaveA-W** [aazuspan/taskee#11]
I'm running Python via a Miniconda3 environment, and I need to query the batch operations that were run under specific service account credentials.  e.g., this works:
```python
import ee
ee.Initialize(ee.ServiceAccountCredentials('...@appspot.gserviceaccount.com', 'secret.json'))
ee.data.listOperations()
```

How can I set credentials for `taskee` to use, or run it from Python interactive after my own call to ee.Initialize()?

**Comment by aazuspan** [comment]
@DaveA-W Thanks for the question. There's currently no way to use anything other than the default persistent credentials with `taskee`, but I'm working on adding this functionality in now.

My plan is to add a `--private_key` option in the CLI, and to allow passing in a custom credential with the Python API. I'll close this issue once that's released in the next few days.

**Comment by aazuspan** [comment]
Using service credentials is now possible in the CLI and API. That will be released in `0.1.0` in the next few weeks, but can be installed immediately using: 

```bash
pip install git+https://github.com/aazuspan/taskee --force-reinstall
```

To use service credentials in the CLI, just provide the relative path to your credentials JSON with the `--private-key` option, e.g.:

```bash
taskee start log --private-key ./secret.json
```

It's not really designed to be run interactively, but if you prefer to use the Python API this should also work:

```py
import ee
from taskee.taskee import Taskee
from taskee.cli.commands import log

credentials = ee.ServiceAccountCredentials(key_file="secret.json")
t = Taskee(credentials=credentials)
log.start(t)
```

I haven't done any thorough testing with a service account, so please let me know if you run into any issues!

**Comment by DaveA-W** [comment]
Works a treat!  Thanks for the quick turnaround on the enhancement :)
